sdk/storage/azblob/container/models.go

```diff
@@ -397,3 +397,31 @@ type SubmitBatchOptions struct {
 func (o *SubmitBatchOptions) format() *generated.ContainerClientSubmitBatchOptions {
 	return nil
 }
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+// FilterBlobsOptions provides set of options for Client.FindBlobsByTags.
+type FilterBlobsOptions struct {
+	// A string value that identifies the portion of the list of containers to be returned with the next listing operation. The
+	// operation returns the NextMarker value within the response body if the listing
+	// operation did not return all containers remaining to be listed with the current page. The NextMarker value can be used
+	// as the value for the marker parameter in a subsequent call to request the next
+	// page of list items. The marker value is opaque to the client.
+	Marker *string
+	// Specifies the maximum number of containers to return. If the request does not specify maxresults, or specifies a value
+	// greater than 5000, the server will return up to 5000 items. Note that if the
+	// listing operation crosses a partition boundary, then the service will return a continuation token for retrieving the remainder
+	// of the results. For this reason, it is possible that the service will
+	// return fewer results than specified by maxresults, or than the default of 5000.
+	MaxResults *int32
```

Comment:
`Include` should also be added in the options?
https://github.com/Azure/azure-sdk-for-go/blob/feature/azblob-stg85/86/sdk/storage/azblob/internal/generated/zz_models.go#L878-L880

Reply:
Is the link to the auto-generated internal type?  I think there might be a typo in the swagger, the "include" parameter is not in the public rest docs - https://learn.microsoft.com/en-us/rest/api/storageservices/find-blobs-by-tags-container?tabs=azure-ad

Reply:
The link is to the auto-generated internal type where `Include` is present in the options. The [request creation code](https://github.com/Azure/azure-sdk-for-go/blob/feature/azblob-stg85/86/sdk/storage/azblob/internal/generated/zz_container_client.go#L515-L517) also has this in the query parameter. 
If it is a typo in the swagger, then this comment can be resolved.

Reply:
I believe there is an error in the swagger because the documentation doesn't include `Include` and it is not present in the the .NET or Python SDK. Will resolve this comment!